### DISCUSSION TITLE
A question about the file param.json of dpgen.

### DISCUSSION THREAD

**Discussion by crsipmax** [deepmodeling/dpgen#1028] [Q&A]
At exploration section of param.json, I want to run several classical molecular dynamics on a particular system,for example:
step 1:The system was equailibrated at 300K for 2000 timesteps.
step 2:Increase tempture from 300K to 5000K within 2000 timesteps.
step 3:Equailibrate at 5000K for 2000 timesteps.
step 4:Decrease tempture from 5000K tom 300K within 2000 timesteps.
step 5:Equailibrate at 300K for 2000 timesteps.
In order to achieve the above purpose,how should I set the "temps" and "nsteps" of "model_devi_jobs" in param.json?The "model_devi_jobs" looks like this:
 "model_devi_jobs": [
        {
            "sys_idx":                      [0],
            "temps":                        [300],
            "press":                        [1.0],
            "trj_freq":                     10,
            "nsteps":                       1000,
            "ensemble":                     "nvt",
            "_idx":                         "00"
        }
    ],
I don't know how to fill the "temps" and "nsteps"，i tconfused me for a long time.

**Comment by HuangJiameng** [comment]
https://docs.deepmodeling.com/projects/dpgen/en/devel/run/param.html Hope this can help you.
